fix(tests): delete flaky test

punya opened the pull request:
This test is already disabled for current and future versions of NodeJS,
and I don't think it's worth putting in the effort to fix flakiness when
the whole test will soon be a no-op.

Also fix other issues that were (deterministically) breaking the build:
* Disable codecov: we were running out of API quota, and most other GCP repos had previously disabled it for unrelated reasons
* Migrate eslint ignores from the "node" plugin to the "n" plugin 

Fixes #1562, fixes #1525, fixes #1524, fixes #1523, fixes #1522, fixes #1500, fixes #1499, fixes #1498, fixes #1497, fixes #1496

punya committed:
fix(tests): delete flaky test

This test is already disabled for current and future versions of NodeJS,
and I don't think it's worth putting in the effort to fix flakiness when
the whole test will soon be a no-op.

aabmass reviewed (approved):
> Fixes #1562, #1525, #1524, #1523, #1522, #1500, #1499, #1498, #1497, #1496

I don't think github is parsing this correctly anymore, I only see one of them under "Development".

punya committed:
fix(tests): work around type check failures

punya committed:
fix(test): disable codecov

Codecov uploads are being throttled (429) because
we are exceeding monthly quota

punya committed:
fix: migrate eslint ignores
